certified-connectors/AgilePoint NX/apiProperties.json

```diff
@@ -0,0 +1,58 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "username": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Username",
+          "description": "The Username for this api",
+          "tooltip": "Provide the Username",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": true,
+            "required": "true"
+          }
+        }
+      },
+      "password": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Password",
+          "description": "The Password for this api",
+          "tooltip": "Provide the Password",
+          "constraints": {
+            "tabIndex": 3,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      },
+      "agilepointServerUrl": {
+        "type": "string",
+        "uiDefinition": {
+          "displayName": "AgilePoint NX Server URL (REST)",
+          "description": "The AgilePoint Server URL",
+          "tooltip": "Provide Your AgilePoint Server URL",
+          "constraints": {
+            "tabIndex": 4,
+            "clearText": true,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",

```

Comment:
Unfortunately the default icon brand color `#007ee5` is not allowed, as well as pure white `#ffffff`.

Reply:
I have updated the color code.